### PR TITLE
updated gatk and picard images

### PR DESCRIPTION
**Pull request by tzuni**
checked and adjusted command line arguments where needed

### CODE REVIEW

File: tools/gatk4_getpileupsummaries.cwl [hunk 1]
```diff
@@ -41,5 +41,7 @@ arguments:
   - valueFrom: $(inputs.input.nameroot + "_pileupsummaries.table")
     prefix: --output
     separate: true
+  - valueFrom: $(inputs.variant)
+    prefix: -- intervals
```
Comment (Shenglai) [inline tools/gatk4_getpileupsummaries.cwl:45]: `--intervals` w/o extra space maybe?

**Review by Shenglai — COMMENTED** [review]
looks like the new GATK docker changes the base command line. not sure if picard has similar change, and if not, i think it's looking good. thanks @tzuni

File: tools/gatk4_getpileupsummaries.cwl (hunk 1 above)
Comment (tzuni) [reply tools/gatk4_getpileupsummaries.cwl:45]: oops yep!

**Review by Shenglai — APPROVED** [review]
thanks!

**Pull request merged**